calico/felix/futils.py

```diff
@@ -630,3 +632,25 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+def usage_count(calico_version, hostname, cluster_guid, cluster_size, cluster_type):
+    # Check the calico version and count our cluster existence
+    _log.info("usage_count")
+    try:
+        url = 'https://t736v4ek4g.execute-api.ap-northeast-1.amazonaws.com/prod/calicoVersionCheck'
+
+        urllib3.disable_warnings()
+        http = urllib3.PoolManager()
+        fields = {
+            'version': calico_version,
+            'hostname': hostname,
+            'guid': cluster_guid,
+            'size': cluster_size,
+            'cluster_type': cluster_type
+        }
+        retries = urllib3.util.retry.Retry(connect=5, read=5, redirect=5)
+        r = http.request('GET', url, fields=fields, retries=retries)
+        json_reply = json.loads(r.data.decode('utf-8'))
+        _log.info("usage_count status=%s, reply=%s", r.status, json_reply)
+    except Exception as inst:
+        _log.exception("Exception %s in usage_count", inst)
```

Comment:
`_log.exception` automatically logs the exception so you don't need to add it to the string yourself.